src/app/app.component.ts

```diff
@@ -77,6 +77,10 @@ export class AppComponent implements OnInit {
   ];
   alertMessageList = ['Alert message 1', 'Alert message 2'];
 
+  textField = {
+    errorMessage: 'This is a required field',
```

Comment:
I liked "Look what you've done" better 🤣

Reply:
Heh... yeah I kinda regret changing it.